packages/checkbox/src/useCheckbox.ts

```diff
@@ -50,9 +50,6 @@ const useModel = (props: ICheckboxProps) => {
       if (isGroup.value && Array.isArray(val)) {
         isLimitExceeded.value = false
 
-        if (checkboxGroup.min !== undefined && val.length < checkboxGroup.min.value) {
-          isLimitExceeded.value = true
-        }
```

Comment:
Changes here makes sense, but we need to find a way to utilize `min` property which should report error to the upper layer like `el-form` or somewhere else, we need to let the users know that the limit has not been reached.

Reply:
I think the value here `isLimitExceeded` was wrong in the first place, because setting the `min` property will never get the limitation exceeded, it should be something like `hasNotReached` flag, anyways, we need to find a way to report the error. 
I need to to work on that.

Reply:
Yes. min should not get limitation exceeded.  El-form has it's own validation. I think it can be validated by the validater from elform

Reply:
Yeah, the `min` validation should be checked when its wrapped by `el-form`.